lib/vector-buffers/src/variants/disk_v2/tests/size_limits.rs

```diff
@@ -59,7 +59,6 @@ async fn writer_error_when_record_is_over_the_limit() {
 }
 
 #[tokio::test]
-#[ignore] // hanging indefinitely
```

Comment:
🎉

Reply:
Oh, woops.  This should not be in the PR.  I uncommented it while building a binary to spin up a background loop of running it until it hung for debugging purposes. 😅 

I'll re-add this.